snuba/query/query_settings.py

```diff
@@ -165,3 +180,9 @@ def get_resource_quota(self) -> Optional[ResourceQuota]:
 
     def set_resource_quota(self, quota: ResourceQuota) -> None:
         pass
+
+    def get_clickhouse_settings(self) -> MutableMapping[str, Any]:
+        return {}
+
+    def set_clickhouse_settings(self, settings: MutableMapping[str, Any]) -> None:
+        pass
```

Comment:
why does this not take effect for subscriptions?

Reply:
I wasn't sure whether subscriptions needed this feature so I didn't add it, but I'll add it to subscriptions as well since there's no harm in adding it.